__tests__/index.test.js

```diff
@@ -4,6 +4,7 @@ const path = require('path');
 const { build } = require('esbuild');
 const fse = require('fs-extra');
 
+const defaultComposer = require('./defaultComposer');
```

Comment:
This function is rather complex and pushes towards the case when one would need a test for a test-only code, which isn't ideal. I assume this is a copy paste from lodash? I think it's fine to add it as a dev dependency and use it directly, that would make tests simpler.

Reply:
Thanks for the feedback. This function is a copy past from the library [default-composer](https://www.npmjs.com/package/default-composer). I updated the pull request and I added it on dev dependency.  I didn't added lodash as it is 1.41mb. This default-composer library is only 41kb.

Reply:
The size mostly doesn't matter for dev deps, especially as they won't be installed when consuming this package. It's more about understating if the function provided was used correctly in the test (so either mock, something well - known or extremely simple). Anyway, let's move on